rpc/core/tx.go

```diff
@@ -20,9 +20,7 @@ import (
 // Tx allows you to query the transaction results. `nil` could mean the
 // transaction is in the mempool, invalidated, or was not sent in the first
 // place.
-// NOTE: proveTx isn't respected but is left in the function signature to
-// conform to the endpoint exposed by Tendermint
-// More: https://docs.tendermint.com/master/rpc/#/Info/tx
+// More: https://docs.tendermint.com/v0.34/rpc/#/Info/tx
```

Comment:
I confirmed the "isn't respected..." statement isn't present at https://github.com/tendermint/tendermint/blob/d32df22c8b43e87e6323b7b1bd114e6e82ed7a27/rpc/core/tx.go#L16-L20